src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -227,6 +227,22 @@ def update(
         ORMRun.get(name).update(display_name=display_name, description=description, tags=tags, **kwargs)
         return self.get(name)
 
+    @monitor_operation(activity_name="pf.runs.delete", activity_type=ActivityType.PUBLICAPI)
+    def delete(
+        self,
+        name: str,
+    ) -> None:
+        """Delete run permanently.
+
+        :param names: run names to delete
+        :param kwargs: other fields to update, fields not supported will be directly dropped.
+        :return: None
+        """
+        run_name = Run._validate_and_return_run_name(name)
+        valid_run = self.get(run_name)
+        LocalStorageOperations(valid_run).delete()
```

Comment:
I don't understand, do we need to construct `LocalStorageOperations` and then call delete function?
Can we directly:
`Shutil. rmtree (path=valid_run. _run. _output_path, oneerror=on_mmtree error)`?

Reply:
We should not get the run outputs folder by pass the LocalStorageOperations.
This is somehow a modular design.